examples/lang/calls/src/lib.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018-2019 Parity Technologies (UK) Ltd.
+// This file is part of ink!.
+//
+// ink! is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// ink! is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with ink!.  If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg_attr(not(any(test, feature = "test-env")), no_std)]
+
+use ink_core::{
+    env::DefaultSrmlTypes,
+};
+use ink_lang::contract;
+use ink_core::env::calls;
+
+contract! {
+    #![env = ink_core::env::DefaultSrmlTypes]
+
+    /// This simple dummy contract dispatches substrate runtime calls
+    struct Calls {}
+
+    impl Deploy for Calls {
+        fn deploy(&mut self) {
+        }
+    }
+
+    impl Calls {
+        /// Dispatches a `transfer` call to the Balances srml module
+        pub(external) fn balance_transfer(&mut self, dest: AccountId, value: Balance) {
```

Comment:
```suggestion
        pub(external) fn transfer_balance(&mut self, dest: AccountId, value: Balance) {
```

Reply:
Moved to https://github.com/paritytech/ink-types-node-runtime/pull/3, not implemented though because we agreed in chat that this name was okay for the `Call`